.github/workflows/make_api_responses.yml

```diff
@@ -0,0 +1,141 @@
+        run: |
+          docker stop neurobagel_node-api-1
+          curl -s http://localhost:8080/query | jq . > api-responses/fapi_query_fail_207.json
+
+      - name: Get recipes for Test node 2
+        uses: actions/checkout@v4
+        with:
+          repository: neurobagel/recipes
+          path: test_node2/recipes
+
+      - name: Configure Test node 2
+        working-directory: test_node2/recipes
+        run: |
+          cp template.env .env
+
+          # Set environment variables
+          sed -i 's|^#\?[[:space:]]\?COMPOSE_PROJECT_NAME=.*|COMPOSE_PROJECT_NAME=neurobagel_node2|' .env
+          sed -i 's|^#\?[[:space:]]\?NB_GRAPH_PORT_HOST=.*|NB_GRAPH_PORT_HOST=7201|' .env
+          sed -i 's|^#\?[[:space:]]\?NB_NAPI_PORT_HOST=.*|NB_NAPI_PORT_HOST=8001|' .env
+          sed -i 's|^#\?[[:space:]]\?NB_RETURN_AGG=.*|NB_RETURN_AGG=false|' .env
+          sed -i 's|^#\?[[:space:]]\?COMPOSE_PROFILES=.*|COMPOSE_PROFILES=local_node|' .env
+        
+      - name: Launch both nodes
+        working-directory: test_node2/recipes
+        run: |
+          docker start neurobagel_node-api-1
+          docker compose up -d
+
+          # Wait for graph to finish being set up
+          until [ -f scripts/logs/DEPLOY.log ] && grep -q "Finished setting up the Neurobagel graph backend." scripts/logs/DEPLOY.log; do
+            sleep 1
+          done
+
+          # Add second n-API to the network of the first node, so that the f-API can access the second n-API by container name
+          docker network connect neurobagel_node_default neurobagel_node2-api-1
+      
+      - name: Generate full success result
+        run: |
+          curl -s http://localhost:8080/query | jq . > api-responses/fapi_query_success_200.json
+
+      - name: Generate protected and open n-API results
+        run: |
+          # Test node 1
+          curl -s http://localhost:8000/query | jq . > api-responses/napi_query_aggregated_results.json
+          # Test node 2
+          curl -s http://localhost:8001/query | jq . > api-responses/napi_query_unaggregated_results.json
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v7
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          add-paths: api-responses/*.json
+          commit-message: Update example API response files
+          title: Update example API responses
+          body: |
+            This PR updates the examples in `api-responses` with the latest changes.
+
+            Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action
+          sign-commits: true
+          base: main
+          branch: create-pull-request/update-api-examples
+          labels: _bot
+
+      - name: Cleanup
+        run: |
+          cd test_node1/recipes && docker compose down && cd ../..
+          cd test_node2/recipes && docker compose down && cd ../..
+          sudo rm -rf test_node1 test_node2
```

Comment:
probably good to do, but is there a specific reason you're cleaning up?

Reply:
Mostly for completeness, to replicate what you might do locally - it shouldn't make a big functional difference since the containers/paths won't persist after the runner stops